html/pages/graphs.inc.php

```diff
@@ -71,7 +71,6 @@
 
     print_optionbar_end();
 
-    print_optionbar_start();
```

Comment:
This leaves a rogue `print_optionbar_end()`. Are you sure it needs removing?

Reply:
Yes, the one above this line is for the normal panel header... it seems there was a start() added, but never had an end.... It also changed the mini-graphs in the view to have a black background with the mono theme....

Reply:
let me revisit this.... as I found one way down in the code..... very weird.... and I dont think its needed to be honest....

Reply:
What I mean is if you search for print_optionbar_ in the file, you now have 5 entries where as surely it should be 4 or 6? 3 starts and 3 ends or 2 of each?

Reply:
Should be good now...

Reply:
BEFORE:
![image](https://user-images.githubusercontent.com/16985649/36739351-5c06b52c-1be8-11e8-87a6-803e367d409b.png)

AFTER:
![image](https://user-images.githubusercontent.com/16985649/36739374-6def4330-1be8-11e8-957f-e3410877f07e.png)